satpy/etc/enhancements/generic.yaml

```diff
@@ -198,7 +207,7 @@ enhancements:
     operations:
     - name: stretch
       method: *stretchfun
-      kwargs: {stretch: linear}
+      kwargs: {stretch: crude, min_stretch: 0, max_stretch: 120}
```

Comment:
@mraspaud @adybbroe @pnuu Thoughts on changing the default reflectance scaling to a static 0 to 120?